cypress/integration/hidden-features.spec.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * A test file containing a number of tests on features and bugs that are often forgotten about when testing PRs.
+ * Can and should be expanded in the future.
+ */
+
+// Before running tests, delete Firestore data for the testing email (so the initial onboarding and walkthrough trigger)
+// Then start on landing page, log in to firebase, and visits the dashboard
+// Delete and log in occurs with TEST_UID and test email of the courseplan testing account using functions from the cypress-firebase package
+before('Delete test user data, then visit site and log in', () => {
+  // log the user in
+  cy.visit('localhost:8080/login');
+  cy.login(Cypress.env('TEST_UID'));
+
+  const TEST_EMAIL = 'courseplan.cornelldti.test@gmail.com';
```

Comment:
😍 
Should we refactor this Firestore reset and call it for every integration test?  
Or at least store `TEST_EMAIL` in a constants file?

Reply:
Going to put TEST_EMAIL in a constants file now!

Reply:
I however do not think we want to do the Firestore reset in every integration test because then we have to go through Onboarding/walkthrough every time. It might make sense though to do a more simple data reset (instead of deleting data, set to only required/basic data) so that tests are not reliant on each other. Making an item for this!